src/app/operations/planning/planning.component.ts

```diff
@@ -83,6 +93,28 @@ export class PlanningComponent implements OnInit {
         this.parentAreas[+operasto.id] = names;
       });
     });
+  }
+
+  async displayActionSheet(operation: Operation) {
+    const actionSheet = await this.actionSheetController.create({
+      header: operation.label.name + ' ' + operation.vegetable.variety + ' ' + operation.vegetable.name,
+      buttons: [{
+        text: 'Détails',
+        icon: 'more',
+        handler: () => {
```

Comment:
I think you can directly put `handler: this.goToDetails(operation.id)`

Reply:
i get error when i try, handler : void doesn't exist
https://ionicframework.com/docs/v3/api/components/action-sheet/ActionSheetController/